internal/resources/common/scope/constant.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package commonscope
```

Comment:
Since you have placed it under resources/common , is this something applicable for other resources also?

Reply:
yes like package install resources, so that's why I've kept under common scope.